docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -571,17 +571,20 @@ which allows for routing of metrics generated by registered MetricsSources to co
 that are implemented by Hadoop include file-based logging, Graphite and Ganglia. All metric sources are exposed via JMX
 when using Metrics2.
 
-Previous to Accumulo 1.7.0, JMX endpoints could be exposed in addition to file-based logging of those metrics configured via
-the +accumulo-metrics.xml+ file. This mechanism can still be used by setting +general.legacy.metrics+ to +true+ in +accumulo-site.xml+.
+Previous to Accumulo 1.7.0, metrics could be exposed using JMX endpoints or file-based logging. These metrics can be enabled
```

Comment:
s/Previous/Prior/

This section talks about what you could do before, which implies it's different now. But this dangling implication is never connected to anything which follows. Should we say something like "Since 1.7.0, we also support .... for metrics." to connect the ideas?